### PR TITLE
Add mintTokens rpc

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Add mintTokens rpc

#### Which issue(s) does this PR fixes?:

Fixes part of #48

### CODE REVIEW

**Comment by jingyi2811** [comment]
Still work in progress

**Review by canonbrother — APPROVED** [review]

File: packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  it('should mintTokens', async () => {
+    const data = await client.token.mintTokens('5@DBTC')
+
+    expect(typeof data).toStrictEqual('string')
+    expect(data.length).toStrictEqual(64)
+
+    await container.generate(1)
+
+    const tokenBalances = await client.account.getTokenBalances()
+    expect(tokenBalances[0]).toStrictEqual('5.00000000@1')
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts:53]: need to assert the before/after to ensure the tokenBalances value is working

**Review by canonbrother — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    }
+    await client.token.createToken({ ...defaultMetadata })
+  }
+
+  it('should mintTokens', async () => {
+    let tokenBalances = await client.account.getTokenBalances()
+
+    expect(tokenBalances.length).toStrictEqual(0)
+
+    const data = await client.token.mintTokens('5@DBTC')
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts:49]: expect error while minting nonexistence coin

**Review by canonbrother — CHANGES_REQUESTED** [review]

File: packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts (hunk 2 above)
Comment (jingyi2811) [reply packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts:50]: Screenshot 2021-06-01 at 1 01 43 PM

Unable to get 10. 
105.6647572 - 95.6648398 = 9.9999174.

There should be fee involved.

**Review by canonbrother — APPROVED** [review]

File: packages/jellyfish-api-core/src/category/token.ts [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import { ApiClient } from '../.'
 
+type TokenRegexType = `${string}@${string}`
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/token.ts:3]:
```suggestion
type TokenRegexType = `${number}@${string}`
```

File: website/docs/jellyfish/api/token.md [hunk 4]
```diff
@@ -134,3 +134,20 @@ interface TokenInfo {
...
+## mintTokens
+
+Creates a transaction to mint tokens.
+
+```ts title="client.token.mintTokens()"
+interface token {
+  mintTokens (payload: TokenRegexType, utxos: UTXO[] = []): Promise<string>
+}
+
+type TokenRegexType = `${string}@${string}`
```
Comment (fuxingloh) [inline website/docs/jellyfish/api/token.md:147]:
```suggestion
type TokenRegexType = `${number}@${string}`
```

File: packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    await expect(promise).rejects.toThrow(RpcApiError)
+    await expect(promise).rejects.toThrow('Invalid Defi token: ETH\', code: 0, method: minttokens')
+  })
+
+  it('should mintTokens with utxos', async () => {
+    let tokenBalances = await client.account.getTokenBalances()
+
+    expect(tokenBalances.length).toStrictEqual(1)
+
+    const { txid } = await container.fundAddress(from, 10)
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts:75]:
```suggestion
    const { txid, vout } = await container.fundAddress(from, 10)
    const inputs: UTXO[] = [{txid, vout}]
```

File: packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+    const utxos = await container.call('listunspent')
+    const inputs: UTXO[] = utxos.filter((utxo: UTXO) => utxo.txid === txid).map((utxo: UTXO) => {
+      return {
+        txid: utxo.txid,
+        vout: utxo.vout
+      }
+    })
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts:83]:
```suggestion
```

**Review by fuxingloh — CHANGES_REQUESTED** [review]

**Review by canonbrother — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**